llvm/test/tools/llvm-dwp/X86/simple.test

```diff
@@ -3,6 +3,14 @@ RUN: llvm-dwarfdump -v %t | FileCheck --check-prefixes=CHECK,NOTYP %s
 RUN: llvm-objdump -h %t | FileCheck --check-prefix=NOTYPOBJ %s
 RUN: llvm-dwp %p/../Inputs/simple/types/a.dwo %p/../Inputs/simple/types/b.dwo -o - \
 RUN:   | llvm-dwarfdump -v - | FileCheck --check-prefixes=CHECK,TYPES %s
+RUN: llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t     --continue-on-cu-index-overflow
+RUN: llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t     --continue-on-cu-index-overflow=continue
+RUN: llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t     --continue-on-cu-index-overflow=soft-stop
+RUN: not llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t --continue-on-cu-index-overflow=foobar
+RUN: llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t     -continue-on-cu-index-overflow
+RUN: llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t     -continue-on-cu-index-overflow=continue
+RUN: llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t     -continue-on-cu-index-overflow=soft-stop
+RUN: not llvm-dwp %p/../Inputs/simple/notypes/a.dwo %p/../Inputs/simple/notypes/b.dwo -o %t -continue-on-cu-index-overflow=foobar
```

Comment:
does this check the behavior? Looks like it checks if the `llvm-dwp` invocation succeeds or fails, but nothing about what it does. So it's unclear/untested that, for instance, `--continue-on-cu-index-overflow` does the `continue` thing, and not something else successful?

Could you add these checks wherever the `--continue-on-cu-index-overflow` behavior is tested, and test them as different phrasings of the same test (eg: there's some test that checks the continue behavior, and it could have two RUN lines, one with `--continue-on-cu-index-overflow` and one with `--continue-on-cu-index-overflow=continue` for instance)

Reply:
Right, it just checks high level functionality. For actual tests. I think it's the same problem as in the original PR. We have to create giant DWP files in our testing environment. Thus it also didn't add any tests: https://github.com/llvm/llvm-project/pull/71902#issuecomment-1828921512